docs/GDI-SOP_charter.md

```diff
@@ -75,24 +75,61 @@ The following table defines the abbreviations and terms relevant to GDI SOPs.
 | **Approver** | TBD                 |               |
 
 ## 4. Introduction
-The [**European Genomic Data Infrastructure (GDI)**](https://gdi.onemilliongenomes.eu) project is a pioneering initiative aimed at **enabling access to genomic, phenotypic, and clinical data across Europe**. This access is crucial for advancing research, policymaking, and healthcare on a continental scale. Central to this mission is the 1+ Million Genomes (1+MG) network, which encompasses a diverse array of national human data-sharing projects, each at different stages of development.
+The [**European Genomic Data Infrastructure (GDI)**](https://gdi.onemilliongenomes.eu) project is a pioneering initiative aims to **enable access access to genomic, phenotypic, and clinical data across Europe**. This access is crucial for advancing research, policymaking, and healthcare on a continental scale. Central to this mission is the 1+ Million Genomes (1+MG) network, which encompasses a diverse array of national human data-sharing projects, each at different stages of development.
```

Comment:
The original version was fine. The updated version has two **access access** in the first sentence and lacks a word between **initiative** _and_ **aims**. 

If we want to keep the updated version I suggest the following: The [**European Genomic Data Infrastructure (GDI)**](https://gdi.onemilliongenomes.eu) project is a pioneering initiative _**and**_ aims to **enable access ~~access~~ to genomic, phenotypic, and clinical data across Europe**.

Reply:
Thanks for spotting. I'm working on a small laptop and editing the texts in here is pretty cumbersome. Sorry!